flags/tests/test_urls.py

```diff
@@ -0,0 +1,146 @@
+from django.conf.urls import url, include
+from django.core.urlresolvers import resolve
+from django.http import HttpResponse, Http404
+from django.test import TestCase, RequestFactory, override_settings
+
+from flags.urls import flagged_url
+from flags.models import Flag
+
+
+def view(request):
+    return HttpResponse('view')
+
+
+def fallback(request):
+    return HttpResponse('fallback')
+
+
+urlpatterns = [
+    flagged_url('FLAGGED_URL', r'^url-true-no-fallback$', view,
+                name='some-view', condition=True),
+    flagged_url('FLAGGED_URL', r'^url-false-no-fallback$', view,
+                name='some-view', condition=False),
+    flagged_url('FLAGGED_URL', r'^url-true-fallback$', view,
+                name='some-view', condition=True, fallback=fallback),
+    flagged_url('FLAGGED_URL', r'^url-false-fallback$', view,
+                name='some-view', condition=False, fallback=fallback),
+]
+
+
+@override_settings(ROOT_URLCONF=__name__)
+class FlagCheckTestCase(TestCase):
+    def setUp(self):
+        self.flag_name = 'FLAGGED_URL'
+        self.factory = RequestFactory()
+
+    def test_flagged_url_true_no_fallback(self):
+        Flag.objects.create(key=self.flag_name, enabled_by_default=True)
+
+        request = self.factory.get('/url-true-no-fallback')
+        resolved_view, args, kwargs = resolve('/url-true-no-fallback')
+        response = resolved_view(request)
+
+        if isinstance(response.content, str):
+            content = response.content
+        else:
+            content = bytes.decode(response.content)
+
+        self.assertEqual(content, 'view')
```

Comment:
Pro-tip: you can use [`self.assertContains`](https://docs.djangoproject.com/en/1.8/topics/testing/tools/#django.test.SimpleTestCase.assertContains) to check the contents of responses without having to manually render.